include/fields2cover/route_planning/route_planner_base.h

```diff
@@ -32,7 +32,7 @@ class RoutePlannerBase {
   /// @param d_tol Tolerance distance to consider if two points are the same.
   /// @return Route that covers all the swaths
   virtual F2CRoute genRoute(
-      const F2CCells& cells, const F2CSwathsByCells& swaths_by_cells,
+      const F2CCells& cells, const F2CSwathsByCells& swaths_by_cells, bool redirect_swaths = true,
```

Comment:
A user that was running `genRoute(cells, swaths, false)` will result in a different behavior now. Instead of breaking the compatibility with previous actual version, create a new function.